googletest/test/googletest-port-test.cc

```diff
@@ -90,10 +90,10 @@ TEST(IsXDigitTest, ReturnsFalseForWideNonAscii) {
 
 class Base {
  public:
-  // Copy constructor and assignment operator do exactly what we need, so we
-  // use them.
   Base() : member_(0) {}
   explicit Base(int n) : member_(n) {}
+  Base(const Base&) = default;
```

Comment:
Are these two lines of change related to the removal of `GTEST_DISALLOW_ASSIGN_`?

Reply:
They're in a separate commit, and related to -Wdeprecated.